src/content/getting-started/quickstart/managed-kubernetes/iks/_index.md

```diff
@@ -0,0 +1,220 @@
+---
+date: 2022-07-03T10:00:00+02:00
+title: "IBM Cloud (IKS)"
+weight: 10
+---
+
+This quickstart guide covers deploying Submariner on two IBM Cloud Kubernetes (IKS) clusters.
+The *Automated example* section point to an automated installer for those who just want to have a quick demo,
+while the *Manual guide* walks through the more important settings and steps.
+
+{{% notice info %}}
+The guide assumes clusters have non-overlapping Pod and Service CIDRs.
+[Globalnet](https://submariner.io/getting-started/architecture/globalnet/) can be used if overlapping CIDRs can't be avoided.
+{{% /notice %}}
+
+## Automated example
+
+The following repo's `submariner-quickstart` [folder](https://github.com/IBM-Cloud/kube-samples/tree/master/submariner-quickstart)
+contains an automated installer that:
+
+- creates two IKS clusters with Terraform
+- installs Submariner on them
+- verifies the setup
+
+For more details, see the repo.
+
+## Manual guide
+
+A step-by-step walkthrough about using Submariner on IBM Cloud.
+
+### Clusters
+
+The exact process of cluster creation is not covered here.
+One can use the [web UI](https://cloud.ibm.com/docs/containers?topic=containers-getting-started&interface=ui),
+the [CLI](https://cloud.ibm.com/docs/containers?topic=containers-getting-started&interface=cli) or
+[Terraform](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest)
+([example](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/container_cluster#vpc-generation-2-ibm-cloud-kubernetes-service-cluster))
+to create IBM Cloud Kubernetes Clusters.
+
+#### Example topology
+
+The guide the following example parameters:
+
+| Cluster Name | Pod CIDR       | Service CIDR  |
+|--------------|----------------|---------------|
+| primary      | 172.17.0.0/18  | 172.21.0.0/16 |
+| secondary    | 172.17.64.0/18 | 172.22.0.0/16 |
+
+### Submariner deployment
+
+#### subctl binary
+
+{{% subctl-install %}}
+
+#### Access
+
+At the time of writing, `subctl` does not support OIDC-based authentication,
+so accessing the clusters can be done through [Service Accounts](https://kubernetes.io/docs/concepts/security/service-accounts/).
+
+The fastest way to obtain a suitable kube config is to run the following command:
+
+```shell
+ibmcloud ks cluster config --admin --cluster CLUSTER_NAME
+```
+
+The example uses `$primary_ctx` and `$secondary_ctx` environment variables.
+Their values can be obtained using `kubectl` like:
+
+```shell
+kubectl config get-contexts -o name
+export primary_ctx=CLUSTER1_NAME
+export secondary_ctx=CLUSTER2_NAME
+```
+
+#### Calico CNI
```

Comment:
Thanks for this PR @adrianrobotka 

Better point to Calico CNI section in Submariner website , as extra IPPools creation and Calico overlay configuration, are common requirements for all platforms running Calico CNI.



[1] 
https://submariner.io/operations/deployment/calico/